pkg/linode-bs/deploy/kubernetes/06-ss-csi-linode-controller.yaml

```
@@ -70,8 +85,18 @@ spec:
                   key: token
           imagePullPolicy: "Always"
           volumeMounts:
+            - name: linode-info
+              mountPath: /linode-info
+            - name: get-linode-id
+              mountPath: /scripts
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
       volumes:
         - name: socket-dir
           emptyDir: {}
+        - name: linode-info
+          emptyDir: {}
+        - name: get-linode-id
+          configMap:
+            name: get-linode-id
+            defaultMode: 493
```

I think these are always worth commenting

```suggestion
            # octal mode 755
            defaultMode: 493
```